bot/exts/moderation/infraction/management.py

```diff
@@ -11,7 +11,9 @@
 
 from bot import constants
 from bot.bot import Bot
-from bot.converters import Expiry, Infraction, MemberOrUser, Snowflake, UnambiguousUser, allowed_strings
+from bot.converters import (
+    Expiry, Infraction, MemberOrUser, Snowflake, UnambiguousUser, allowed_strings
+)
```

Comment:
Since you removed the import you don't need to break the line anymore

Reply:
Yep, fixed :+1: